mailbox/elasticsearch-v7/src/test/java/org/apache/james/mailbox/elasticsearch/v7/ElasticSearchIntegrationTest.java

```diff
@@ -129,6 +128,7 @@ protected void initializeMailboxManager() throws Exception {
     }
 
     @Test
+	@Disabled("Types cannot be provided in get mapping requests, unless include_type_name is set to true.")
```

Comment:
We tend to follow the convention to put @Test and @Disabled at the same indentation level. Any reason to be doing some different?